astroquery/heasarc/core.py

```diff
@@ -228,7 +245,10 @@ def _parse_result(self, response, verbose=False):
             table = Table.read(data, hdu=1)
             return table
         except ValueError:
-            return self._fallback(response.content)
+            try:
+                return self._fallback(response.content)
```

Comment:
following the note above - maybe this should be `response.text`?

Reply:
ok, done.